.travis.yml

```diff
@@ -50,7 +50,7 @@ matrix:
     - php: 7.2
       env: TEST_CONFIG="phpunit.xml"
     - php: 7.2
-      env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml" SYMFONY_VERSION="~2.8.0"
```

Comment:
1. We can drop testing of elastic if we want, it's deprecated
2. We should do this for all cases of `SYMFONY_VERSION `

Reply:
About 1: I don't know the whole picture (whether we will introduce Elastic one day and will need the code again?). But if I understand correctly - if this job fails one day (because of real reasons, not like in this PR) we won't be fixing it? If so then it might be a good idea to remove it.

About 2: Wasn't sure about it as it's set to 2.7.x (instead of 2.8.x) for Postgresql job. Should it be set to `~2.8.46` in every job?

I can create a follow up with both things, just need @alongosz opinion as well.

Reply:
1. This is my opinion, but afaik It will resurface as external package. Most of the code will probably be redone, maybe on Elastic Bundle for instance (partner requested), and as the current prototype is stuck on ElasticSearch 1.x so needs quite a lot of work to get it working on newer versions anyway.

2.´2.8.46´  => yes, we removed support for 2.7 when we did the composer cleanup recently afaik